geth/account/accounts_test.go

```diff
@@ -109,3 +111,46 @@ func TestVerifyAccountPasswordWithAccountBeforeEIP55(t *testing.T) {
 	_, err = acctManager.VerifyAccountPassword(keyStoreDir, address.Hex(), TestConfig.Account3.Password)
 	require.NoError(t, err)
 }
+
+func TestCreateAndRecoverAccount(t *testing.T) {
+	ctrl := gomock.NewController(t)
+	nodeManager := common.NewMockNodeManager(ctrl)
+
+	// Create an arbitrary keystore for the process
+	keyStoreDir, err := ioutil.TempDir(os.TempDir(), "accounts")
+	require.NoError(t, err)
+	defer os.RemoveAll(keyStoreDir) //nolint: errcheck
+	keyStore := keystore.NewKeyStore(keyStoreDir, 4, 5)
+
+	manager := account.NewManager(nodeManager)
+	password := "some-pass"
+
+	// Fail if account keystore can't be acquired
```

Comment:
What do you think about extracting failing scenarios to a separate test? Something like `TestCreateAccountFailure()` that could verify this error?

Reply:
Ah yes, makes more sense. Thanks.